index.js

```diff
@@ -100,23 +75,19 @@ app.get('/category/:ctyId', (req, res) => {
     categoryId = `cty${userInput.padStart(2, '0')}`
   }
 
-  // Getting category name
-  for (let category of categoryData.categories) {
-    if (category.id === categoryId) {
-      categoryNameVal = category.categoryName;
-      break;
-    }
-  }
+  // Creating required Maps
+  dataService.getCategories;
+  dataService.getCombinedProductMap
```

Comment:
These are not needed. `getCombinedProductMap`'s return value is not used.

Reply:
Correct, I think I accidentally added it